migrations/sql/V2019.09.19.02.53__add_historical_mine_incident_followup_type_code.sql

```diff
@@ -0,0 +1,16 @@
+INSERT INTO mine_incident_followup_investigation_type
+	(
+	mine_incident_followup_investigation_type_code,
+	description,
+	display_order,
+    active_ind,
+	create_user,
+	update_user
+	)
+VALUES
+('HUK', 'Historical - Unknown', 40, 'F', 'system-mds', 'system-mds')
+ON CONFLICT DO NOTHING;
+
+ALTER TABLE mine_incident ADD COLUMN mms_min_acc_no character varying(9); 
```

Comment:
I disagree with verbatim putting mms column names into our data structure. We should have an ETL_INCIDENTS table if we need to persist key relationships, and if we need to transform data we should have our own column names.